test/promises_aplus_adapter.js

```diff
@@ -13,11 +14,9 @@ require( "jsdom" ).env( "", function ( errors, window ) {
 		var deferred = jQuery.Deferred();
 
 		return {
-			get promise() {
-				return deferred.promise();
-			},
+			promise: deferred.promise(),
```

Comment:
It doesn't require a getter, it requires a property. Using a getter works as it de facto changes a method to a property but this works as well; tests pass (this is tested on Travis so it wouldn't be green if it failed in any tested Node/io.js).